### PR TITLE
Accordion for human footprint

### PR DESCRIPTION
**Pull request by alegnaaived**
It has three template components in the subfolder "humanFootprint"

### CODE REVIEW

File: src/humanFootprint/CurrentFootprint.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import RenderGraph from '../charts/RenderGraph';
+
+const CurrentFootprint = geofence => (
```
Comment (erikasv) [inline src/humanFootprint/CurrentFootprint.jsx:4]: the parameter of the function must be an object for properties (props), and hence geofence must be one property of that parameter

File: src/humanFootprint/CurrentFootprint.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          area: 521758, percentage: 0.30040794096373092685, type: 'Alta', color: '#b3433b',
+        }],
+      'Tipo de área', 'Comparación', 'SmallBarStackGraph',
+      'Cobertura', null, null, null,
+      'Estado de la cobertura en el área seleccionada', '%')}
+    </div>
+  </div>
+);
+
+export default CurrentFootprint;
```
Comment (erikasv) [inline src/humanFootprint/CurrentFootprint.jsx:38]: missing propTypes

File: src/humanFootprint/PersistenceHumanFootprint.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import RenderGraph from '../charts/RenderGraph';
+
+const PersistenceFootprint = geofence => (
```
Comment (erikasv) [inline src/humanFootprint/PersistenceHumanFootprint.jsx:4]: Same as in CurrentFootprint

File: src/humanFootprint/PersistenceHumanFootprint.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          area: 1054399, percentage: 0.5674571823442289, type: 'Estable alta', color: '#b3433b',
+        }],
+      'Tipo de área', 'Comparación', 'SmallBarStackGraph',
+      'Cobertura', null, null, null,
+      'Estado de la cobertura en el área seleccionada', '%')}
+    </div>
+  </div>
+);
+
+export default PersistenceFootprint;
```
Comment (erikasv) [inline src/humanFootprint/PersistenceHumanFootprint.jsx:36]: also missing proptypes

File: src/humanFootprint/TimelineFootprint.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const TimelineFootprint = geofence => (
```
Comment (erikasv) [inline src/humanFootprint/TimelineFootprint.jsx:3]: Same as in CurrentFootprint

File: src/humanFootprint/TimelineFootprint.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      </h4>
+      <h4>
+        Páramo
+      </h4>
+      Área del ecosistema dentro de la unidad de consulta: 332 ha
+    </div>
+  </div>
+);
+
+export default TimelineFootprint;
```
Comment (erikasv) [inline src/humanFootprint/TimelineFootprint.jsx:37]: also missing PropTypes

File: src/search/HumanFootprint.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import AddIcon from '@material-ui/icons/Add';
+import Accordion from '../commons/Accordion';
+import CurrentFootprint from '../humanFootprint/CurrentFootprint';
+import TimelineFootprint from '../humanFootprint/TimelineFootprint';
+import PersistenceFooprint from '../humanFootprint/PersistenceHumanFootprint';
+
+const HumanFootprint = (geofence) => {
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:8]: Same as in CurrentFootprint

File: src/search/HumanFootprint.jsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  const componentsArray = [{
+    label: {
+      id: 'Huella humana actual',
+      name: 'Huella humana en el último año',
+      disabled: false,
+      expandIcon: <AddIcon />,
+      detailId: 'Huella humana actual en área de consulta',
+      description: 'Huella humana identificada en el último año de medición disponible, sobre el área de consulta',
+    },
+    component: CurrentFootprint(geofence),
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:18]: When CurrentFootprint is fixed, this call must be as a component, not as a function

File: src/search/HumanFootprint.jsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  {
+    label: {
+      id: 'Huella humana en el tiempo',
+      name: 'Huella humana a través del tiempo',
+      disabled: false,
+      expandIcon: <AddIcon />,
+      detailId: 'Huella humana a través del tiempo en el área',
+      description: 'Representa diferentes análisis de huella humana en esta área de consulta',
+    },
+    component: TimelineFootprint(geofence),
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:29]: When TimelineFootprint is fixed, this call must be as a component, not as a function

File: src/search/HumanFootprint.jsx [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+  {
+    label: {
+      id: 'Persistencia de huella humana',
+      name: 'Persistencia',
+      disabled: false,
+      expandIcon: <AddIcon />,
+      detailId: 'Persistencia de la huella humana en la unidad de consulta',
+      description: 'Representa la persistencia desde el origen del muestreo hasta el periodo actual, producto de análisis de huella humana en el tiempo y en esta área de consulta',
+    },
+    component: PersistenceFooprint(geofence),
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:40]: When PersistenceFooprint is fixed, this call must be as a component, not as a function

File: src/search/HumanFootprint.jsx [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <div>
+      <Accordion
+        componentsArray={componentsArray}
+      />
+    </div>
+  );
+};
+
+HumanFootprint.propTypes = {
```
Comment (erikasv) [inline src/search/HumanFootprint.jsx:52]: geofence must be one property, also I think it should be required, so no defaultProps in this case.

Comment (alegnaaived) [reply src/search/HumanFootprint.jsx:53]: Done

**Pull request merged**